pom.xml

```diff
@@ -402,7 +448,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.4.2</version>
+                <version>3.2.1</version>
```

Comment:
What was the old shaded jar size versus the new size ?

Reply:
This is to add support for Hadoop 3 to presto so presto works when environment has Hadoop 3. 

Currently, some queries fail when it runs on Hadoop 3. It is very likely that once we start supporting Hadoop 3 things will start failing for Hadoop 2, so the recommendation is to create separate repository to support Hadoop 3.

Reply:
I am not an expert on Hadoop, so will defer this to Hadoop experts.